cinema/serializers.py

```diff
@@ -1 +1,103 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import (
+    CinemaHall,
+    Genre,
+    Actor,
+    Movie,
+    MovieSession
+)
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+    capacity = serializers.IntegerField(read_only=True)
+
+    class Meta:
+        model = CinemaHall
+        fields = (
+            "id",
+            "name",
+            "rows",
+            "seats_in_row",
+            "capacity"
+        )
+
+    def get_capacity(self, obj):
+        return obj.rows * obj.seats_in_row
```

Comment:
The `get_capacity` method is unnecessary because the `capacity` property is already defined in the `CinemaHall` model. You can remove this method and rely on the `capacity` property directly.